### PR TITLE
MdaExtractors + Ironclust

### PR DESCRIPTION
**Pull request by alejoe91**
- [x] tests

### CODE REVIEW

File: spikeinterface/extractors/mdaextractors.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+        self._diskreadmda = DiskReadMda(str(self._timeseries_path))
+        dtype = self._diskreadmda.dt()
+        num_channels = self._diskreadmda.N1()
+        assert geom.shape[0] == self._diskreadmda.N1(), f'Incompatible dimensions between geom.csv and timeseries ' \
+                                                        f'file: {geom.shape[0]} <> {self._diskreadmda.N1()}'
+        BaseRecording.__init__(self, sampling_frequency=self._dataset_params['samplerate'] * 1.0,
+                               channel_ids=np.arange(num_channels), dtype=dtype)
+        rec_segment = MdaRecordingSegment(self._diskreadmda)
+        self.add_recording_segment(rec_segment)
+        self.set_channel_locations(geom)
```
Comment (samuelgarcia) [inline spikeinterface/extractors/mdaextractors.py:36]: We should create a probe object here no ?

File: spikeinterface/extractors/utils/mdaio.py [hunk 2]
```diff
@@ -0,0 +1,544 @@
+import numpy as np
```
Comment (samuelgarcia) [inline spikeinterface/extractors/utils/mdaio.py:1]: I would copy/paste all that function inside the mdaextractors.
Even if it make a super file. So evrything is centralise and more easy to manage.
No other module will depend on this mdaio.
Am I wring ?

File: spikeinterface/extractors/mdaextractors.py (hunk 1 above)
Comment (alejoe91) [reply spikeinterface/extractors/mdaextractors.py:36]: possibly, but we don't have default electrode info. We should have a function that makes a probe out of locations. I think it should be the default behavior in set_locations

File: spikeinterface/core/testing.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import numpy as np
+
+
+def check_recordings_equal(RX1, RX2, segment_idx=0, return_scaled=True, force_dtype=None, check_times=True):
```
Comment (samuelgarcia) [inline spikeinterface/core/testing.py:4]: Here I would make a loop internaly to all segments!!

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**